app/containers/HomeContainer/index.js

```diff
@@ -106,28 +107,22 @@ export function HomeContainer({
             ))}
           </Skeleton>
         </CustomCard>
-      )
+      </If>
     );
   };
   const renderErrorState = () => {
-    let repoError;
-    if (reposError) {
-      repoError = reposError;
-    } else if (!get(reposData, 'totalCount', 0)) {
-      repoError = 'respo_search_default';
-    }
+    const repoError = reposError || (!get(reposData, 'totalCount', 0) ? 'respo_search_default' : null);
+
     return (
-      !loading &&
-      repoError && (
+      <If condition={!loading && repoError}>
         <CustomCard color={reposError ? 'red' : 'grey'} title={intl.formatMessage({ id: 'repo_list' })}>
           <T id={repoError} />
         </CustomCard>
-      )
+      </If>
     );
   };
   const refreshPage = () => {
```

Comment:
since this no longer  a `Refresh` button we can name it something relevant

Reply:
👍